NetSuiteProject/Exercise.js

```diff
@@ -2,22 +2,65 @@
 function add(x,y){
 	return x + y;
 };
+
+function addition(x,y){
+	if(typeof add (x,y) === "string"){
+	alert("This function only accepts Integers")
+	}
+	else {return x + y;}
+}
+
+
 //subtract
 function subtract(x, y){
 	return (x - y);
 };
+
+function subtraction(x, y){
+	if(isNaN(subtract(x,y))  === true){
+	alert("This function only accepts Integers");
+	}
+	else {return (x - y);}
+}
+
+
 //Multiplication
 function multiply(x, y){
 	return (x * y);
 };
+
+function multiplication(x,y){
+	if(isNaN(multiply(x,y)) === true ){
+		alert("This function only accepts Integers");
+	}
+	else {return (x * y);}
+}
+
+
 //Division
 function divide (x, y){
 	return (x / y);
 };
+
+function division(x, y){
+	if(isNaN(divide(x,y)) === true){
+		alert("This function only accepts Integers");
```

Comment:
I never worked on server-side so that part I'm not sure. I only know the pop up boxes that will show on the browser. Never really experienced server-side scripting. I have experience working within a database such as Microsoft SQL but I never knew or learned how to bridge the gap to create the communication channel between user, server, and database.